first_mpi_class/Hector_Rivas_MPI_Class.py

```diff
@@ -0,0 +1,18 @@
+""" this file is to manage all examples about MPI using the mpi4py librery """
+
+from mpi4py import MPI
+
+"""MPI class"""
+class MPI_rank():
+    def __init__(self):
+        self.mpi = None
+    
+    """ This method gets the rank """
+    def method_for_rank(self):
+        rank = MPI.COMM_WORLD.Get_rank()
+        print('Rank: ', rank)
```

Comment:
Avoid prints, return the Rank, and Log the printed message using the Python Logging Library